python/simulation.py

```diff
@@ -960,7 +960,6 @@ def __init__(self,
                  filename_prefix=None,
                  output_volume=None,
                  output_single_precision=False,
-                 load_structure='',
```

Comment:
Also need to update the docstrings for the function `load_structure`:

https://github.com/NanoComp/meep/blob/9509547b997747d9ba331c3a60a2508c8d918c67/python/simulation.py#L1871-L1878

Note that changing the docstrings requires building the markdown page `Python_User_Interface.md` for the user manual as described in https://github.com/NanoComp/meep/blob/master/doc/README.md in the paragraph starting with "To (re)generate the Python API documentation..."

The documentation in `Python_User_Interface.md.in` also needs to be updated:

https://github.com/NanoComp/meep/blob/9509547b997747d9ba331c3a60a2508c8d918c67/doc/docs/Python_User_Interface.md.in#L350-L373

Reply:
done.